sympy/core/power.py

```diff
@@ -162,7 +162,8 @@ def __new__(cls, b, e, evaluate=None):
                 return S.One
             elif e is S.One:
                 return b
-            elif e.is_integer and _coeff_isneg(b):
+            # Only perform autosimplification if base if a Symbol or number
+            elif (b.is_Symbol or b.is_number) and e.is_integer and _coeff_isneg(b):
```

Comment:
Eventually everything in the core should only look at assumptions if it's a number or Symbol, but we are doing things in parts. Does too much break if you apply the rule to `e`?